### PR TITLE
Rename tchap_room_settings_remove_from_directory

### PR DESCRIPTION
**Pull request by Claire1817**
#669 
Screenshot_1660049256
Screenshot_1660205673

### CODE REVIEW

File: vector/src/main/res/values-fr/strings_tchap.xml [hunk 1]
```diff
@@ -52,8 +52,8 @@
     <string name="tchap_scan_media_error_file_is_infected">Le fichier est infecté, impossible de le télécharger.</string>
 
     <!-- tchap room settings -->
-    <string name="tchap_room_settings_remove_from_directory">Retirer ce salon de la liste des forums</string>
-    <string name="tchap_room_settings_remove_from_directory_prompt_msg"><b>Cette action est irréversible.</b>\nVoulez-vous vraiment retirer ce salon de la liste des forums ?</string>
+    <string name="tchap_room_settings_remove_from_directory">Convertir en salon privé (irréversible)</string>
+    <string name="tchap_room_settings_remove_from_directory_prompt_msg"><b>Cette action est irréversible.</b>\nVoulez vous vraiment convertir ce forum en salon privé ?</string>
```
Comment (Florian14) [inline vector/src/main/res/values-fr/strings_tchap.xml:56]:
```suggestion
    <string name="tchap_room_settings_remove_from_directory_prompt_msg"><b>Cette action est irréversible.</b>\nVoulez-vous vraiment convertir ce forum en salon privé ?</string>
```

Comment (Florian14) [inline vector/src/main/res/values-fr/strings_tchap.xml:56]: I think we can also rename the keys that are not anymore related to removing from directory

Comment (giomfo) [reply vector/src/main/res/values-fr/strings_tchap.xml:56]: > I think we can also rename the keys that are not anymore related to removing from directory

@Florian14 @Claire1817 : I suggest to replace "tchap_room_settings_remove_from_directory" with "tchap_room_settings_convert_to_private" in both keys name. Wdyt?

File: vector/src/main/res/values-fr/strings_tchap.xml [hunk 2]
```diff
@@ -52,8 +52,8 @@
     <string name="tchap_scan_media_error_file_is_infected">Le fichier est infecté, impossible de le télécharger.</string>
 
     <!-- tchap room settings -->
-    <string name="tchap_room_settings_remove_from_directory">Retirer ce salon de la liste des forums</string>
-    <string name="tchap_room_settings_remove_from_directory_prompt_msg"><b>Cette action est irréversible.</b>\nVoulez-vous vraiment retirer ce salon de la liste des forums ?</string>
+    <string name="tchap_room_settings_convert_to_private">Retirer ce salon de la liste des forums</string>
+    <string name="tchap_room_settings_convert_to_private_prompt_msg"><b>Cette action est irréversible.</b>\nVoulez-vous vraiment retirer ce salon de la liste des forums ?</string>
```
Comment (Florian14) [inline vector/src/main/res/values-fr/strings_tchap.xml:56]: You have reset the strings to previous ones

File: vector/src/main/res/values/strings_tchap.xml [hunk 3]
```diff
@@ -52,8 +52,8 @@
     <string name="tchap_scan_media_error_file_is_infected">We have detected that this file is infected, so it cannot be downloaded.</string>
 
     <!-- tchap room settings -->
-    <string name="tchap_room_settings_remove_from_directory">Remove this room from the forums directory</string>
-    <string name="tchap_room_settings_remove_from_directory_prompt_msg"><b>This action is irreversible.</b>\nAre you sure you want to remove this room from the forums directory?</string>
+    <string name="tchap_room_settings_convert_to_private">Remove this room from the forums directory</string>
+    <string name="tchap_room_settings_convert_to_private_prompt_msg"><b>This action is irreversible.</b>\nAre you sure you want to remove this room from the forums directory?</string>
```
Comment (Florian14) [inline vector/src/main/res/values/strings_tchap.xml:56]: idem

**Review by yostyle — APPROVED** [review]

**Review by giomfo — APPROVED** [review]
SGTM

**Review by Florian14 — APPROVED** [review]

**Pull request merged**